front/pages/api/v1/w/[wId]/data_sources/[dsId]/tables/csv.ts

```diff
@@ -0,0 +1,79 @@
+import type { NextApiRequest, NextApiResponse } from "next";
```

Comment:
git diff seems to have missed the rename here; assuming the changes were the same typical ones as above